runtimevar/blobvar/blobvar.go

```diff
@@ -16,6 +16,19 @@
 // variables read from a blob.Bucket.
 // Use NewVariable to construct a *runtimevar.Variable.
 //
+// URLs
+//
+// For runtimevar.OpenVariable URLs, blobvar registers for the scheme "blob".
```

Comment:
The requirement is that schemes are unique per package, not unique globally.

#1453 will make it so that you can also (optionally) add the API to the scheme, e.g. `runtimevar+blob://blahblahblah`, to provide a consistent way of making URLs globally unique and more self-documenting; I think that addresses your concern.